pkg/jsonpb/README.md

```diff
@@ -0,0 +1,15 @@
+This package is copied from github.com/gogo/protobuf at commit deb6fe8ca7c6d06584bfbd40ca407bf69d9fd2aa.
+
+
+## Customizations
+
+1. Serialize `int64` and `uint64` to integers, not strings.
```

Comment:
Is all of this to be able to use `int64` and `uint64`? I think I would argue we just disallow those types rather than attempt to maintain our own marshallers and unmarshallers. Just wondering how we decided to go down this path

Reply:
as per discussing with @yuval-k we have certain int64's in gloo that we can't get rid of. discussed and determined that the cost of maintaining a fork of `jsonpb` would be minimal